web/src/main/java/se/inera/intyg/minaintyg/monitoring/UserConverter.java

```diff
@@ -0,0 +1,37 @@
+package se.inera.intyg.minaintyg.monitoring;
+
+import ch.qos.logback.classic.pattern.ClassicConverter;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import java.util.Objects;
+import org.springframework.security.core.context.SecurityContextHolder;
+import se.inera.intyg.minaintyg.auth.MinaIntygUser;
+import se.inera.intyg.minaintyg.util.HashUtility;
+
+public class UserConverter extends ClassicConverter {
+
+  @Override
+  public String convert(ILoggingEvent iLoggingEvent) {
+    return userInfo();
+  }
+
+  private String userInfo() {
+    final var minaIntygUser = minaIntygUser();
+    if (minaIntygUser == null) {
+      return "noUser";
+    }
+    return HashUtility.hash(minaIntygUser.getPersonId()) + "," + minaIntygUser.getLoginMethod();
+  }
+
+  private MinaIntygUser minaIntygUser() {
```

Comment:
Move to a static "toolkit" to get the logged in User. Maybe consider to return a Optional.